sql/api/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -4263,6 +4263,14 @@ object functions {
   def sentences(string: Column, language: Column, country: Column): Column =
     Column.fn("sentences", string, language, country)
 
+  /**
+   * Splits a string into arrays of sentences, where each sentence is an array of words.
+   * @group string_funcs
+   * @since 4.0.0
+   */
+  def sentences(string: Column, language: Column): Column =
```

Comment:
Maybe adding such parameter usage in SQL functions would align the two sides.

Reply:
This LGTM

Reply:
We should also add it into Scala Spark Connect client, and Python implementation as well.

Reply:
oh..., yes, I overlooked it, let me make it up together, thanks!

Reply:
- `Python` side:
https://github.com/apache/spark/blob/32b054c40602c7355176903fa32224774f0c1bec/python/pyspark/sql/functions/builtin.py#L11282-L11287

- `Python Connect` side:
https://github.com/apache/spark/blob/32b054c40602c7355176903fa32224774f0c1bec/python/pyspark/sql/connect/functions/builtin.py#L2470-L2478

I think `Python` and `Python Connect` are already supported. I have updated the `description` of this function and added `new test cases`.

Reply:
Based on my understanding, after [PR](https://github.com/apache/spark/pull/47839), we only need to add `new functions` in `sql/api/src/main/scala/org/apache/spark/sql/functions.scala`, and both `Scala` and `Scala Connect` should be supported. 
Do I understand correctly?